src/styles/App/_events-by-category-page.scss

```diff
@@ -0,0 +1,87 @@
+.events-by-category-page {
+  > h2 {
+    font-size: 2.3rem;
+  }
+  .events-list {
+    .event-card {
+      background-color: #fff;
+      width: calc(70% - 30px);
+      height: 190px;
+      list-style: none;
+      box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.05);
+      margin: 15px auto 0 auto;
+
+      > a {
+        text-decoration: none;
+        color: inherit;
+
+        .event-container {
+          display: flex;
+          height: 100%;
+
+          .event-header {
+            display: flex;
+            flex-direction: column;
+            width: 300px;
+
+            .event-image {
+              width: 300px;
+              height: 150px;
+              background-color: #cf6664;
+
+              > img {
+                max-height: 100%;
+                max-width: 100%;
+              }
+            }
```

Comment:
careful with your indentation, try to not go more than 3 indentations inside a parent, it may give you trouble for making it responsive and may be difficult if you make any mark-up changes!
All of your element have classes, try to use the classes instead of the indentation.